crates/bitcoind_rpc/tests/test_emitter.rs

```diff
@@ -744,7 +624,7 @@ fn mempool_during_reorg() -> anyhow::Result<()> {
         // emission.
         // TODO: How can have have reorg logic in `TestEnv` NOT blacklast old blocks first?
```

Comment:
@LagginTimes general question: after this PR is merged, do you think we'll be able to solve this TODO ?

Reply:
Unfortunately no. This can be a feature added in a follow up!